data/test/scenarios/overlay_text_demo.cfg

```diff
@@ -0,0 +1,162 @@
+#textdomain wesnoth-test
+
+[test]
+    name = _ "Overlay Text Demo"
+    map_file=test/maps/generic_unit_test.map
+    id = overlay_text_demo
+    turns = unlimited
+    
+    [side]
+        side=1
+        controller = human
+        [leader]
+            name = _ "Alice"
+            type = Elvish Archer
+            id=alice
+        [/leader]
+    [/side]
+    [side]
+        side=2
+        controller = human
+        [leader]
+            name = _ "Bob"
+            type = Orcish Grunt
+            id=bob
+        [/leader]
+    [/side]
+    [event]
+        name=preload
+        [lua]
+            code=<<
+                left_label_id = {valid = false}
+                right_label_id = left_label_id
+            >>
+        [/lua]
+    [/event]
+    [label]
+        x,y=3,8
+        text=_"Remove label left"
+        tooltip=_"Removes the text on the left side of the screen"
+    [/label]
+    [event]
+        name=moveto
+        first_time_only=no
+        [filter]
+            x=3,3
+            y=6,8
+        [/filter]
+        [lua]
+            code=<<
+                if left_label_id.valid then
+                    left_label_id:remove()
+                end
+            >>
+        [/lua]
+    [/event]
+    [label]
+        x,y=15,8
+        text=_"Remove text right"
+        tooltip=_"Removes the text on the right side of the screen"
+    [/label]
+    [event]
+        name=moveto
+        first_time_only=no
+        [filter]
+            x=15,15
+            y=6,8
+        [/filter]
+        [lua]
+            code=<<
+                if right_label_id.valid then
+                    right_label_id:remove()
+                end
+            >>
+        [/lua]
+    [/event]
+    [label]
+        x,y=3,6
+        text=_"Add text left"
+        tooltip=_"Adds text on the left side of the screen"
+    [/label]
+    [event]
+        name=moveto
+        first_time_only=no
+        [filter]
+            x,y=3,6
+        [/filter]
+        [lua]
+            code=<<
+                left_label_id = wesnoth.interface.add_overlay_text(
+                    {"Left Label", size = 42, color = "ff8000"},
+                    {"unlimited", 500},
+                    -100, -60
+                )
+            >>
+        [/lua]
+    [/event]
+    [label]
+        x,y=15,6
+        text=_"Add text right"
+        tooltip=_"Adds text on the right side of the screen"
+    [/label]
+    [event]
+        name=moveto
+        first_time_only=no
+        [filter]
+            x,y=15,6
+        [/filter]
+        [lua]
+            code=<<
+                right_label_id = wesnoth.interface.add_overlay_text(
+                    {"Right Label", size = 42, color = {0, 128, 255}},
+                    {"unlimited", 500},
```

Comment:
This line doesn't set a duration, although either of the following would work here:
* `"unlimited",`
* `{duration = "unlimited", 500},`